src/mapml.js

```diff
@@ -598,7 +601,113 @@ window.M = M;
     }
   });
 }());
-M.Util = {
+M.Util = {  
+  extractInputBounds: function(template){
+    if(!template) return undefined;
+    let inputs = template.values, projection = template.projection || FALLBACK_PROJECTION, value = 0, boundsUnit = FALLBACK_CS;
+    let bounds = L.bounds(L.point(0,0),L.point(5,5)), nMinZoom = 0, nMaxZoom = this[projection].options.resolutions.length - 1;
```

Comment:
This seems odd to have a particular  bounds in here.

Reply:
Good catch, it was supposed to be updated to `...crs.tilematrix.bounds(0);`, it's used as a fallback incase bounds cannot be derived from the template